features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/CustomPluginNotifierDescriptionScenario.m

```diff
@@ -0,0 +1,46 @@
+#import "CustomPluginNotifierDescriptionScenario.h"
+#import <Bugsnag/BugsnagPlugin.h>
+
+@interface Bugsnag()
++ (id)notifier;
++ (void)registerPlugin:(id<BugsnagPlugin>)plugin;
+@end
+
+@interface DescriptionPlugin : NSObject<BugsnagPlugin>
+@property (nonatomic, getter=isStarted) BOOL started;
+@end
+
+@implementation DescriptionPlugin
+
+- (instancetype)init {
+    self = [super init];
+    return self;
+}
+
+- (void)start {
+    id notifier = [Bugsnag notifier];
```

Comment:
Could we consider passing `notifier` as a parameter to `start`? This would avoid each plugin implementation requiring knowledge about `Bugsnag.h`

Reply:
The intent was to keep the interface internal for now. If we want to open it up later as a feature, then it would make sense to add it to a public header instead. As it is, access to use is gated by internal APIs, and the library is vendored, so there's no chance of the API breaking without also breaking the build process.

Reply:
Regardless, I think it'd be simpler for our own internal plugin implementations if we could just pass the notifier in as a parameter. Not too fussed on this point so I'll leave it up to you.

Reply:
The notifier is a private class, and `[Bugsnag notifier]` is private (not in any header). Even if it was passed in, the interface for notifier would have to come from somewhere, so passing it in wouldn't fix this issue.

Reply:
Ah got it - I had assumed `BugsnagNotifier` was visible. This approach makes sense.